### DISCUSSION TITLE
Snapshot publishing

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/bloop#1952] [Q&A]
So quick question on how we're doing snapshot publishing. Right now publishing both snapshots and releases to sonatype releases:

```
1.5.5
1.5.5-2-9298eee4
1.5.5-4-247774ae
1.5.6
1.5.6-6-aed542c7
1.5.6-11-caf93adf
1.5.6-14-0520499d
1.5.6-16-fa25f778
1.5.6-18-07d4d70d
1.5.6-20-dcd104ad
1.5.6-22-f1159f10
1.5.6-24-bd164e70
1.5.6-26-04e37730
1.5.6-29-b2f5ecd9
1.5.6-31-db2757ec
1.5.6-33-3a69be45
1.5.6-35-f736b2a5
```

As we migrate to [sbt-ci-release](https://github.com/sbt/sbt-ci-release) by default this will instead change the versioning so that snapshots actually end with `-SNAPSHOT` and are published to sonatype snapshots. I'm personally in favor of doing this that way as I don't like the way we currently pollute releases with snapshots, but this does change how we are currently doing this. Before I make this change I want to ensure it's ok. The impact that it may have on users is that in order to use the snapshot versions they'll need to add sonatype snapshots to their resolvers.

**Comment by julienrf** [comment]
That change looks very reasonable to me.

**Reply by tgodzik** [reply]
We should also make sure that the launcher uses the sonatype snapshot repository by default, but I think that is a good idea.
